MediaGalleryUi/view/adminhtml/web/template/grid/columns/image.html

```diff
@@ -6,10 +6,12 @@
 -->
 <div class="media-gallery-wrap" collapsible>
     <div class="media-gallery-image">
-        <div class="masonry-image-block media-gallery-image-block" attr="'data-id': $col.getId($row())" css="{ selected: isSelected($row()) }">
+        <div class="masonry-image-block media-gallery-image-block"
+             attr="'data-id': $col.getId($row())"
+             css="{ selected: isSelected($row()) }"
+             click="function(){ if (!$collapsible.opened()) {select($row())} }" >
```

Comment:
Can you please move this function from view to model method? It's not good to have some logic in html template. thanks